sh/openrc-run.sh.in

```diff
@@ -39,6 +39,22 @@ sourcex()
 	fi
 }
 
+apply_ulimits()
+{
+	if [ $(( $# % 2 )) -eq 0 ]; then
+		while [ $# -gt 0 ] ; do
+			flag=$1
+			val=$2
+			shift 2
+			if ! ulimit "${flag}" "${val}"; then
```

Comment:
while usually variable names are nice, here they just seem to make reading a bit harder, considering ulimit is quite simple, we can:
```suggestion
			if ! ulimit "$1" "$2"; then
				eerror "${RC_SVCNAME}: unable to apply RC_ULIMIT settings"
			fi
			shift 2
```